EmpireLegends/GameStart.cpp

```diff
@@ -2,67 +2,88 @@
 #include "MapLoader.h"
 #include "Player.h"
 #include "BiddingFacility.h"
+#include "PlayerStrategies.h"
 #include <iostream>
 
+//Game Engine Commented
 GameStart::GameStart() {
-	mapType = 0;
-	numOfPlayers = 0;
+	//  typeMap = 0;
+	  //numOfPlayers = 0;
+	//players = {};
+	//biddingPlayers = {};
 }
 
 GameStart::~GameStart() {
-	mapType = 0;
-	numOfPlayers = 0;
-	for (Player* player : players) {
+	//typeMap = 0;
+	//numOfPlayers = 0;
+	/*for (Player* player : players) {
 		delete player;
-	}
+	}*/
 }
 
-void GameStart::loadMap() {
+GameMap* GameStart::loadMap() {
 
 	MapLoader* mapLoader = new MapLoader();
-
-	std::cout << "------------------------------" << std::endl;
-	std::cout << "Eight-Minute-Empire: Legends" << std::endl;
-	std::cout << "-----------------------------" << std::endl;
-
+	int x = 0;
 	std::cout << "Please enter the type of map you wish to play with." << std::endl;
 	std::cout << "	- Enter 1 for a rectangular map (4 continents)." << std::endl;
 	std::cout << "	- Enter 2 for an L-shaped map (3 continents)." << std::endl;
 	std::cout << "	- Enter 3 for a T-shaped map (4 continents)." << std::endl;
-	std::cin >> mapType;
+	std::cin >>x;
 
+	while (std::cin.fail()) {
+		std::cin.clear();
+		std::cin.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
+		std::cout << "Bad entry.  Enter a NUMBER: ";
+		std::cin >> x;
+	}
+
+	//typeMap = x;
```

Comment:
think it might be better to keep `int mapType` and replace your instances of `x` with it..?